portal/tests.py

```diff
@@ -0,0 +1,35 @@
+from django.test import RequestFactory
+from django.contrib.sessions.backends.db import SessionStore
+from django.urls import reverse
+
+import pytest
+from portal import views
+
+
+@pytest.mark.django_db
+def test_index():
+    # Get correct request
+    request = RequestFactory().get(reverse("portal:index"))
+
+    # Create session
+    s = SessionStore()
+
+    # Add session to request
+    request.session = s
+
+    # Get response. Since index is a function, this is the correct way
+    response = views.index(request)
+
+    # Check if it returns the correct status code
+    assert response.status_code == 200
+
+
+def test_HomeView():
```

Comment:
I think this is more in line with pep8

```suggestion
def test_home_vew_class():
```

Reply:
I agree, but with `view` instead of `vew` hehe